plugins/rapture/nexus-rapture-plugin/src/main/resources/static/rapture/NX/controller/User.js

```diff
@@ -248,10 +248,14 @@ Ext.define('NX.controller.User', {
           win.options.success.call(win.options.scope, win.options);
         }
       },
-      failure: function() {
+      failure: function(response) {
+        var message = NX.I18n.get('GLOBAL_SERVER_INCORRECT_CREDENTIALS_WARNING');
```

Comment:
opinionated change here as disconnect would still allow signin attempts, which was hard-wired to tell you that you had wrong user/pass